pkg/kubelet/eviction/helpers.go

```diff
@@ -746,14 +746,14 @@ func makeSignalObservations(summaryProvider stats.SummaryProvider, capacityProvi
 				diskUsageP := &diskUsage
```

Comment:
I think here we should use the similar as you did in eviction manager to calculate ephemeralStorageAllocatable. That is more clear

if *m.dedicatedImageFs {
 +		fsStatsSet = []fsStatsType{fsStatsLogs, fsStatsLocalVolumeSource}
 +	} else {
 +		fsStatsSet = []fsStatsType{fsStatsRoot, fsStatsLogs, fsStatsLocalVolumeSource}
 +	}

Reply:
OK, done